fileio/ft_read_event.m

```diff
@@ -514,6 +514,10 @@
               event(end  ).value    = tok{2};
               event(end  ).sample   = str2num(tok{3});
               event(end  ).duration = str2num(tok{4});
+              if size(tok, 2) >= 6
```

Comment:
please change this into `if size(tok, 2) >= 6 && ft_platform_supports('datetime')` to make it conditional on the MATLAB version

Reply:
and can you please add the check for datetime to `fieldtrip/utilities/ft_platform_supports.m`?

Reply:
Done

Reply:
Check for datetime also done